src/test/kotlin/org/stellar/walletsdk/helpers/objectFromJsonFile.kt

```diff
@@ -0,0 +1,15 @@
+package org.stellar.walletsdk.helpers
+
+import com.google.gson.Gson
+import java.io.File
+
+fun <T> objectFromJsonFile(filePath: String, t: Class<T>): T {
+  val jsonString =
+    try {
+      File(filePath).readText()
+    } catch (e: Exception) {
+      throw Error("File not found")
+    }
+
+  return Gson().fromJson(jsonString, t)
```

Comment:
This creates a Gson() object every time we call.  The anchor SDK uses the[ singleton pattern t](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/core/src/main/java/org/stellar/anchor/util/GsonUtils.java#L19)o avoid the problem and provides a single point of access if we want to[ inject adapters](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/core/src/main/java/org/stellar/anchor/util/GsonUtils.java#L15) not supported by Gson.